include/util.js

```diff
@@ -705,7 +705,8 @@ Util.getExtension = function(filePath, options) {
     
     //do to the end of the path
     var pathPartIndex = filePath.lastIndexOf(options.sep || path.sep) || 0;
-    filePath = filePath.substr(pathPartIndex);
+    if (pathPartIndex > -1)
+    	filePath = filePath.substr(pathPartIndex);
```

Comment:
Would you mind including brackets with the if statement?  Coveralls yells at us over it.